fedimint-server-ui/src/dashboard.rs

```diff
@@ -28,7 +28,7 @@ pub fn dashboard_layout(content: Markup) -> Markup {
                 (layout::common_head("Dashboard"))
             }
             body {
-                div class="container" style="max-width: 66%;" {
```

Comment:
@joschisan Generally we should avoid inline styles, and try to stick with boostrap feel just for simplicity sake, but I'm not sure if AI is going to understand it.